### DISCUSSION TITLE
迅雷云盘在晚上会被限速，是否取的链接是限速链接？

### DISCUSSION THREAD

**Discussion by woniu871** [AlistGo/alist#3134] [Q&A]
返回的JSON 数据包中 ， web_content_link 字段表示 普通用户的限速下载地址 ， 通常情况下限速 1M/S
links.application/octet-stream 表示不限速的 VIP 下载地址
一般在晚上会被限300KB-500KB左右，视频卡卡的。

**Comment by anwen-anyi** [comment]
迅雷貌似晚上就是会限速吧毕竟用的人多了  dddd，阿里似乎也会也是晚上用的人多了

**Reply by woniu871** [reply]
但是我晚上用pc的迅雷和手机的迅雷播放视频是没限速哦的，白金账号，alist里登录的也是那账号

**Comment by wiskiiii** [comment]
不是因为人多限速，刚挂载好的时候很快，过段时间就一直是1M/s以下了

**Reply by woniu871** [reply]
并不是哦，不信你白天用，速度没问题的，我的意思是希望作者能给有会员账号的用会员的链接，这样就不会晚上限速

**Comment by kuko7** [comment]
我的迅雷使用Infuse播放，播放十几秒就卡住然后就不能继续了，是什么原因呀？都能正常查看文件的

**Comment by woniu871** [comment]
我的问题应该是解决了，用专业版配置

**Reply by anwen-anyi** [reply]
普通版的默认的走代理模式，搭建Alist机器的宽带不够速度也上不去...
专业版就适合自己调整哈哈  普通版就是无脑挂 给不会配置的用户用 哈哈

**Comment by jonawang019** [comment]
我专业版现在晚上还是出现慢的问题，网页端似乎没问题，infuse就不行，本地路由都检查过了不知道啥原因
